sdk/storage/azblob/service/client.go

```diff
@@ -82,6 +83,23 @@ func NewClientFromConnectionString(connectionString string, options *ClientOptio
 	return NewClientWithNoCredential(parsed.ServiceURL, options)
 }
 
+// GetUserDelegationCredential obtains a UserDelegationKey object using the base ServiceURL object.
+// OAuth is required for this call, as well as any role that can delegate access to the storage account.
+func (s *Client) GetUserDelegationCredential(ctx context.Context, info KeyInfo, o *GetUserDelegationKeyOptions) (*UserDelegationCredential, error) {
```

Comment:
The options type name is slightly different, it should match the method name with the "Options" suffix.

Reply:
Sorry not the parameter name (what you had there was fine).  It's the type name that needs to be renamed to `GetUserDelegationCredentialOptions`

Reply:
All good! I updated it now, thank you for clarifying :)